packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol

```diff
@@ -758,9 +828,11 @@ contract L1StandardBridge_FinalizeBridgeETH_Test is Bridge_Initializer {
 
         l1StandardBridge.finalizeBridgeETH{ value: 100 }(alice, alice, 100, hex"");
     }
+}
 
-    /// @dev Tests that finalizing bridged ETH reverts if a custom gas token is used.
-    function test_finalizeBridgeETH_isCustomGasToken_reverts() external {
+contract L1StandardBridge_FinalizeBridgeETH_TestFail is Bridge_Initializer {
+    /// @dev Tests that finalizing bridged reverts with custom gas token.
+    function test_finalizeBridgeETH_customGasToken_reverts() external {
         vm.mockCall(
             address(l1StandardBridge.messenger()),
             abi.encodeWithSelector(CrossDomainMessenger.xDomainMessageSender.selector),
```

Comment:
This could be a good fuzz test

Reply:
I thought we'd just revert for all values with custom gas token? if so, I don't think it'd make sense to fuzz?